constants/links.ts

```diff
@@ -43,6 +43,9 @@ export const EXTERNAL_LINKS = {
 		Optimism: 'https://optimistic.etherscan.io/tx',
 		OptimismKovan: 'https://kovan-optimistic.etherscan.io/tx',
 	},
+	Optimism: {
+		Home: 'https://www.optimism.io/',
```

Comment:
Just a nitpick, but I think we can remove the www:
```suggestion
		Home: 'https://optimism.io/',
```

Reply:
Updated.